ansible/Vagrantfile.Debian10

```diff
@@ -7,7 +7,7 @@ sudo apt-get install software-properties-common -y
 sudo apt-get install gpg -y	
 sudo apt-get update
 # Get IPs of the VM, and output to shared folder
-ip address show | grep -e "\\binet\\b" | cut -d' ' -f6 | cut -d/ -f1  >> /vagrant/playbooks/AdoptOpenJDK_Unix_Playbook/hosts.tmp
+ip address show | grep -e "\\inet\\b" | cut -d' ' -f6 | cut -d/ -f1  >> /vagrant/playbooks/AdoptOpenJDK_Unix_Playbook/hosts.tmp
```

Comment:
DId you mean to remove this `b`?

Reply:
Weirdly the give the same output - I'll change it back to keep it consistent with the Deb8 VF